benches/group.rs

```diff
@@ -0,0 +1,51 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use ff::Field;
+use group::prime::PrimeCurveAffine;
+use halo2curves::secp256k1::Secp256k1;
+use pasta_curves::arithmetic::{CurveAffine, CurveExt};
+use rand_core::OsRng;
+
+fn criterion_benchmark<G: CurveExt>(c: &mut Criterion) {
+    // G1Projective
+    {
+        let name = "GProjective";
+        let a = G::generator();
+        let a_affine = G::AffineExt::generator();
```

Comment:
Should we use some random value here? Since the old algorithm seems to have some optimization for adding the same point (https://github.com/privacy-scaling-explorations/halo2curves/blob/093c4dc12bee6ca54a0254534ba2d6497f3b3d67/src/derive/curve.rs#L965 and https://github.com/privacy-scaling-explorations/halo2curves/blob/093c4dc12bee6ca54a0254534ba2d6497f3b3d67/src/derive/curve.rs#L1006)